.github/workflows/docker.yaml

```diff
@@ -76,7 +76,7 @@ jobs:
       - name: Build and push
         uses: docker/build-push-action@v4
         with:
-          platforms: linux/amd64
+          platforms: linux/amd64, linux/arm64
```

Comment:
Hmm yeah, I think that's what used to fail before as well when I disabled these in the first place. Is there a way to skip only that orch-tester build, since it's not something we're using/needing ARM image anyway?

Reply:
changed the workflow to skip arm64 builds for orch-tester.